libpromises/conversion.c

```diff
@@ -351,80 +351,41 @@ DataType ConstraintSyntaxGetDataType(const ConstraintSyntax *body_syntax, const
 
 /****************************************************************************/
 
+// Warning: Defaults to true on unexpected (non-bool) input
 bool BooleanFromString(const char *s)
```

Comment:
These two functions could now be `inline`, but no biggie.

Reply:
Maybe we should add some rules on when to inline in [CONTRIBUTING.md](https://github.com/cfengine/core/blob/master/CONTRIBUTING.md)?

Reply:
It's part of the craft. :wink: